data/org.freedesktop.portal.Settings.xml

```diff
@@ -62,6 +62,14 @@
 
       Unknown values should be treated as ``0`` (no preference).
 
+    * ``org.freedesktop.appearance``  ``cursor-theme`` (``s``)
+
+      Indicates the system's preferred cursor theme.
+
+    * ``org.freedesktop.appearance``  ``cursor-theme-size`` (``u``)
+
+      Indicates the system's preferred cursor theme size.
```

Comment:
It would be good to mention if this includes output scaling (how it's expected to handle Hi-DPI / fractional-scaling) for e.g.

Reply:
Indeed, I would assume it be in "logical pixels" like size, meaning equivalent to the surface size one would set with `wl_pointer.set_cursor`, which is more or less what the existing cursor theme size settings do, and what `gsettings get org.gnome.desktop.interface cursor-size` would return.

Reply:
This could be mentioned in the description then (or a reference elsewhere - to more detailed docs).
On my own desktop I've noticed the cursor size jumping erratically between windows. So having a convention for calculating the final cursor size would be good (including rounding behavior with fractional scaling).
Although that doesn't have to be handled in this PR, failure to define this anywhere likely means cursor size will "jump" from a user perspective.